central/auth/service/service_impl_test.go

```diff
@@ -279,9 +247,9 @@ func (s *authServiceAccessControlTestSuite) TestValidateAuthMachineToMachineConf
 				Issuer:                  "https://stackrox.io",
 				Mappings: []*v1.AuthMachineToMachineConfig_Mapping{
 					{
-						Key:   "some-key",
-						Value: "a(b",
-						Role:  testRole1,
+						Key:             "some-key",
+						ValueExpression: "a(b",
```

Comment:
Thinking aloud: is empty `ValueExpression: ""` valid or not?

Reply:
Ah, it is seen as valid, and matches _everything_ so we probably should guard for this.